activities/Markdown.activity/js/activity.js

```diff
@@ -24,7 +24,7 @@ define(["sugar-web/activity/activity","sugar-web/datastore"], function (activity
             inputTextContent.value = markdowntext;
             markdownParsing(); //to load again when there is a datastore entry
         });
-
+		document.getElementById("insertText").value = l10n_s.get("insertText");
```

Comment:
- You don't have to change the "value" but the "title" property. So you have to write ".title" instead of ".value"
- Please do the same thing for other buttons in the toolbar (generated by the framework), ids are "wmd-bold-button-second", "wmd-italic-button-second", "wmd-heading-button", "wmd-hr-button", "wmd-olist-button", "wmd-ulist-button", "wmd-code-button", "wmd-quote-button", "wmd-link-button", "wmd-undo-button", "wmd-redo-button"